index.html

```diff
@@ -1,18 +1,64 @@
-<script>
-let name = document.location.pathname.slice(1);
+<!DOCTYPE html>
+<html lang="en">
 
-if (!name) {
-  console.log("no place name specfied");
-} else {
-  console.log("fetching place named '" + name + "'");
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet" />
+  <title>Document</title>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+    integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+    crossorigin="" />
+  <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+    integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+    crossorigin=""></script>
+</head>
 
-  fetch(name + '.json')
-    .then(function (response) {
-      return response.text();
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
+<body>
+  <!--Title Header-->
+  <section class="main-container">
+    <!--Title Container-->
+    <section class="title-flex-container">
+      <p id="yelpington-flex"><a href="http://localhost:5000/" style="text-decoration: none">YELPINGTON...</a></p>
+    </section>
+    <!--Sidebar/Map-->
+    <section class="sidebar-map-container">
+      <!--Sidebar Container-->
+      <section class="sidebar-container">
+        <div id="restaurant-title">Restaurants</div>
+        <div class="restaurants" id="restaurant-list"></div>
+      </section>
+      <!--Map Container-->
+      <section class="map-container">
+        <div id="map"></div>
+        <script src="index.js"></script>
+      </section>
+    </section>
 
-</script>
+    <!--Footer Container-->
+    <section class="footer-container">
+      <!--Footer Title-->
+      <div id="footer-title">
+        Questions or Comments? <strong>Get in touch:</strong>
+      </div>
+      <!--Footer Text Boxes-->
+      <div class="center" id="footer-inputs">
+        <input type="text" id="footer-input-name" placeholder="Name" />
+        <input type="text" id="footer-input-email" placeholder="E-mail" />
+        <textarea type="textarea" id="footer-input-message" placeholder="Message">
+      </textarea>
+        <!--Footer Submit Button-->
+        <button type="button" id="footer-submit-button">Send Message
+        </button>
+      </div>
+      <div class="footer-info">
+        <div id="footer-info-message">
+          A message will go here soon.
+        </div>
+      </div>
+    </section>
+    <script src="main.js"></script>
```

Comment:
This is fine here, try to place all of your script tags as the last tags in your body. (So just swap this line with line 61)